vhdl/vgenQuartus.m

```diff
@@ -27,8 +27,13 @@
 % Bozeman, MT 59718
 % support@flatearthinc.com
 
-function vgenTcl(infile, outfile, additionalFilesetAbsDir)
+function vgenQuartus(infile, outfile, additionalFilesetAbsDir)
```

Comment:
Separated out. I am not sure what you mean by this though:
> rather than having them separate as they have been.

Reply:
Sorry, I wasn't very clear there. The IP core is made up of all the .vhd files and the associated *_hw.tcl file. Currently, the .vhd files are generated by a program, and the _hw.tcl file is generated by a separate program. 

I was thinking it might be nice to have a wrapper program that generates both. For the Simulink workflow, doing so makes no difference at all. But for workflows that don't rely on Simulink, the wrapper program could be nice as it reduces one step. For non-Simulink workflows, users just need to create a dataplane.json file by hand and feed that into the vhdl/tcl/driver scripts to autogenerate code. 

Either way, combing the vhdl and _hw.tcl generation into one program is more a of conceptual consideration than a practical one.